packages/widget.vite-plugin/src/common/validateWidgetConfig.ts

```diff
@@ -75,5 +75,15 @@ function validateWidgetParameters(parameters: ParameterConfig): void {
         `Parameter id "${parameterId}" does not match allowed pattern (must be camelCase)`,
       );
     }
+    if (parameterConfig.type === "objectSet") {
+      if (
+        typeof parameterConfig.objectType.internalDoNotUseMetadata?.rid
+          !== "string"
+      ) {
+        throw new Error(
```

Comment:
Should we include some more detail on parameterConfig / parameterConfig.objectType when this happens so we know exactly what the shape looked like?